import_csv.py

```diff
@@ -195,3 +198,60 @@
 cursor.execute(sql_cnt)
 cnt = cursor.fetchone()
 logger.info(f'  Rows imported into DB: {cnt[0]}')
+
+logger.info('****************************************************************************************************')
+logger.info('Import Grocery stores data')
+
+sql_drop = '''DROP TABLE IF EXISTS grocery_stores;'''
+cursor.execute(sql_drop)
+
+cursor.execute(f'''
+CREATE TABLE grocery_stores(
+  cid varchar(50),
+  category_name varchar(50),
+  lat char(10),
+  lon char(10),
+  day char(2),
+  hour integer,
+  popularity integer
+);
+''')
+
+df_stores_raw = pd.read_csv('/data/raw/dataset_crawler-google-places_2023-05-25_19-49-43-534.csv')
+logger.info(f'df_stores_raw.shape={df_stores_raw.shape}')
+for index, row in df_stores_raw.iterrows():
+    for day in ['Mo', 'Tu', 'We', 'Th', 'Fr', 'Sa', 'Su']:
+        first_hour_column = f'popularTimesHistogram/{day}/0/hour'
+        first_hour_raw = row[first_hour_column]
+        if np.isnan(first_hour_raw):
+            continue
+        first_hour = int(first_hour_raw)
+        for hour in range(0, 24):
+            if hour < first_hour:
+                popularity = 0
```

Comment:
We want to handle situation when hours are ordered in a different way (e.g. hour 4-23 and then 0-3)